### ISSUE TITLE
nucypher-deploy transfer-tokens is not working

### ISSUE THREAD

**Issue by cygnusv** [nucypher/nucypher#1561]
```
Traceback (most recent call last):
  File "/Users/david/.local/share/virtualenvs/nucypher-cjt3uwkK/bin/nucypher-deploy", line 11, in <module>
    load_entry_point('nucypher', 'console_scripts', 'nucypher-deploy')()
  File "/Users/david/.local/share/virtualenvs/nucypher-cjt3uwkK/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/david/.local/share/virtualenvs/nucypher-cjt3uwkK/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/david/.local/share/virtualenvs/nucypher-cjt3uwkK/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/david/.local/share/virtualenvs/nucypher-cjt3uwkK/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/david/.local/share/virtualenvs/nucypher-cjt3uwkK/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/david/NuCypher/dev/nucypher/nucypher/cli/common_options.py", line 143, in wrapper
    return func(**kwargs)
  File "/Users/david/NuCypher/dev/nucypher/nucypher/cli/common_options.py", line 143, in wrapper
    return func(**kwargs)
  File "/Users/david/NuCypher/dev/nucypher/nucypher/cli/deploy.py", line 465, in transfer_tokens
    target_address=target_address)
  File "/Users/david/NuCypher/dev/nucypher/nucypher/blockchain/eth/decorators.py", line 70, in wrapped
    return func(*args, **kwargs)
  File "/Users/david/NuCypher/dev/nucypher/nucypher/blockchain/eth/agents.py", line 179, in transfer
    contract_function = self.contract.functions.transfer(target_address, amount)
  File "/Users/david/.local/share/virtualenvs/nucypher-cjt3uwkK/lib/python3.7/site-packages/web3/contract.py", line 819, in __call__
    clone._set_function_info()
  File "/Users/david/.local/share/virtualenvs/nucypher-cjt3uwkK/lib/python3.7/site-packages/web3/contract.py", line 829, in _set_function_info
    self.kwargs
  File "/Users/david/.local/share/virtualenvs/nucypher-cjt3uwkK/lib/python3.7/site-packages/web3/_utils/contracts.py", line 155, in find_matching_fn_abi
    raise ValidationError(message)
web3.exceptions.ValidationError: 
Could not identify the intended function with name `transfer`, positional argument(s) of type `(<class 'str'>, <class 'nucypher.blockchain.eth.token.NU'>)` and keyword argument(s) of type `{}`.
Found 1 function(s) with the name `transfer`: ['transfer(address,uint256)']
Function invocation failed due to no matching argument types.
```

**Comment by cygnusv** [comment]
I already have a fix for this in a local branch. It's a really minor bug.

**Comment by cygnusv** [comment]
Closed by #1559
